composer.json

```diff
@@ -10,22 +10,21 @@
     ],
     "homepage": "https://github.com/laravel-notification-channels/africastalking",
     "license": "MIT",
-    "authors": [
-        {
-            "name": "Osaigbovo Emmanuel",
-            "email": "osaigbovoemmanuel1@gmail.com",
-            "role": "Developer"
-        }
-    ],
+    "authors": [{
+        "name": "Osaigbovo Emmanuel",
+        "email": "osaigbovoemmanuel1@gmail.com",
+        "role": "Developer"
+    }],
     "require": {
         "php": "^7.2.5",
-        "africastalking/africastalking": "^2.3",
-        "illuminate/notifications": "~5.5 || ~6.0 || ~7.0",
-        "illuminate/support": "~5.5 || ~6.0 || ~7.0"
+        "appslabke/africastalking": "^2.5.1",
+        "guzzlehttp/guzzle": "^7.0.1",
```

Comment:
Can we also allow guzzle 6?

Reply:
Sure lemme include it too

Reply:
Fixed with this commit 44557fb717b282659250de66bdd0ab3274fbdc13